### ISSUE TITLE
Jetpack NUX: Add Existing WordPress

### ISSUE THREAD

**Issue by johnHackworth** [Automattic/wp-calypso#6400]
We’ve previously discussed the idea to test having an “Add Existing WordPress” option to the site switcher.

The route for this already exists: https://wordpress.com/jetpack/connect 

What we need to do here is:
Create the UI/component needed to display a call to action to add an existing site at the bottom of the site switcher.
Link that to the route
Set this up as an a/b test on Calypso (may need help from @breezyskies team) 
 

![image](https://cloud.githubusercontent.com/assets/1554855/16449761/910623a8-3df9-11e6-9ee1-873d6a044a1e.png)

**Comment by johnHackworth** [comment]
@rickybanister what are we going to do with is design-wise? just add another link under the "add new WordPress" link, or something more elaborated?

**Comment by rickybanister** [comment]
I think we will want to explore a few ideas and perhaps test more than one option. 

The two links solution feels very clumsy. Also we've learned that having illustrated instructions images have helped quite a bit, so perhaps we re-explore the idea of a modal with illustrated options. That might give users a better understanding of what both options mean.

**Comment by mtias** [comment]
Yes, please — we should either do a dialog or at least a popover menu on the same button. Probably reword it to "Add WP Site" without the "new".

**Comment by rickybanister** [comment]
Another idea we've thrown around is modifying the signup flow's new Step 1 design with a link:

<img width="614" alt="screen shot 2016-06-29 at 10 59 19 am" src="https://cloud.githubusercontent.com/assets/437258/16457181/914df54a-3de8-11e6-9707-f5748da9e33d.png">

**Comment by johnHackworth** [comment]
Visually, I like it way better with that link inside of the first step of the site creation flow... but wouldn't that be a little hard to discover for users that use wp.com to admin their jetpack sites (so they don't usually "add new sites" from calypso) ?

**Comment by rickybanister** [comment]
I think we will simply need to propose a handful of solutions and test out what we think may work. It's important that we balance which use cases are used more frequently and also discoverability. I don't want it to be too hidden, but also no option has ever existed previously to add a Jetpack site from within WordPress.com. We'll just have to try things out without confusing .com-only users.

**Comment by johnHackworth** [comment]
maybe this is a good task for @tyxla to start and get more familiar with calypso

**Comment by rickybanister** [comment]
More design and coordination needs to happen before we start building anything here. Happy to discuss further.

**Comment by tyxla** [comment]
Recently I had to test something with the Jetpack Connect flow. As a newcomer, I found it rather odd that I should initiate the WordPress.com flow from the Jetpack site. So having this feature within Calypso is a necessity in my opinion.

My two cents: just like @mtias suggested, I think we can remove the "new" from the button, so it would become "Add WordPress" or "Add WordPress site". Clicking on it could display a screen with the following two options: 

1. **Create new WordPress site** - leading to the current survey first step;
2. **Connect a WordPress site** - leading to the JPC flow.

@rickybanister thoughts?

**Comment by johnHackworth** [comment]
I've just discovered that we are changing "add a new WordPress" for "add a new site": 

https://github.com/Automattic/wp-calypso/issues/5722

**Comment by rickybanister** [comment]
@tyxla I agree with you completely, it's just a matter of spending some dedicated time thinking about the experience and also coordinating with other folks to make sure we don't disrupt the WordPress.com new site flows too much. Likely we will have to only launch to a small subset of users and make sure to adequately track any increase in confusion.

**Comment by johnHackworth** [comment]
Should we involve someone else in this task? I think delta should be involved in this (as in "I'm not sure if we are stepping into Delta shoes if we implement this " :) ) 

ping @meremagee

**Comment by rickybanister** [comment]
This little one has been tossed around a bit—the other day while discussing it with @keoshi the idea came up to try using our `popover` component to display a menu with both options. That's the absolute easiest way to get a test in the wild. Can we spin that up to some small percentage of people (or perhaps people that already have at least 1 jetpack site in their sites list?


Mockup:
<img width="304" alt="screen shot 2016-10-05 at 3 51 06 pm" src="https://cloud.githubusercontent.com/assets/437258/19128989/9567b5da-8b13-11e6-8e58-01d4eb2f0fe5.png">

**Comment by mtias** [comment]
Yes, a popover menu was my preference as well. Maybe the first phrasing should read "Start a new WP.com site" and the other "Add an existing self-hosted WordPress site"?

**Comment by johnHackworth** [comment]
:+1: Let's do this and do a small a/b test. The only thing I'm afraid of is no one using the self-hosted option (if you know what is all this about, you probably also know how to use jetpack to link your site with wpcom, and I don't think you are going to go to this link just in case instead of just connecting jetpack from wp-admin)
